crates/macro/ui-tests/wasm-bindgen.rs

```diff
@@ -0,0 +1,43 @@
+extern crate wasm_bindgen as extern_test;
```

Comment:
Ideally we should have a separate crate which renames wasm-bindgen in Cargo.toml but this should also suffice